tests/test_client.py

```diff
@@ -1926,53 +1926,44 @@ def test_update_all_versions_fail_on_bad_metadata(client, user):
         assert version["authz"] == mock_authz_A
 
 
-@pytest.mark.parametrize(
-    "mock_authz_permissions",
-    [
-        {
-            "test": {
-                "create": ["resource_A", "resource_B"],
-                "update": ["resource_A", "resource_B"],
-            },
-            "test_user": {"update": ["resource_B"]},
-        }
-    ],
-)
 def test_update_all_versions_fail_on_missing_permissions(client, user, use_mock_authz):
```

Comment:
i think you don't need the `user` fixture at all since everything is mocked 😅

Reply:
Word, I was keeping it in there as a way to insert the first two setup records. I could also do it with a call to use_mock_authz to allow uploading/creating both record_A and record_B.

Reply:
ah got it